### PR TITLE
dynamips 0.2.23

### PR DESCRIPTION
**Pull request by Porkepix**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/dynamips.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Dynamips < Formula
   desc "Cisco 7200/3600/3725/3745/2600/1700 Router Emulator"
   homepage "https://github.com/GNS3/dynamips"
-  url "https://github.com/GNS3/dynamips/archive/v0.2.22.tar.gz"
-  sha256 "5b3142eb4d2780683ea8781d5f4da6fc39c514d36546392508c74da8ba98240b"
-  license "GPL-2.0"
+  url "https://github.com/GNS3/dynamips/archive/v0.2.23.tar.gz"
+  sha256 "503bbb52c03f91900ea8dbe8bd0b804b76e2e28d0b7242624e0d3c52dda441a1"
+  license "GPL-2.0-or-later"
```
Comment (SMillerDev) [inline Formula/dynamips.rb:6]: Where does it say this? I just created https://github.com/GNS3/dynamips/issues/135 because it doesn't specify

**Comment by iMichka** [comment]
In case upstream does not answer in the next hours, let's maybe just ship this without the license change.

File: Formula/dynamips.rb (hunk 1 above)
Comment (Porkepix) [reply Formula/dynamips.rb:6]: I used this: https://github.com/GNS3/dynamips/blob/master/COPYING#L296-L299 mainly and also this: https://github.com/GNS3/dynamips/blob/master/COPYING#L242-L248

Comment (SMillerDev) [reply Formula/dynamips.rb:6]: That's the standard license text for the GPLv2 license

Comment (SMillerDev) [reply Formula/dynamips.rb:6]: It doesn't specify if later versions can be used though, it just says that upstream can specify if they want to allow that.

Comment (Porkepix) [reply Formula/dynamips.rb:6]: My bad for being mistaken there, then.

Comment (p-linnane) [reply Formula/dynamips.rb:6]: @Porkepix Try this: https://tools.spdx.org/app/check_license/

@chenrui333 Showed it to me and I've been using it extensively.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**